datafusion/common/Cargo.toml

```diff
@@ -41,10 +41,14 @@ pyarrow = ["pyo3", "arrow/pyarrow"]
 [dependencies]
 apache-avro = { version = "0.14", default-features = false, features = ["snappy"], optional = true }
 arrow = { workspace = true, default-features = false }
+arrow-array = { version = "35.0.0", default-features = false, features = ["chrono-tz"] }
```

Comment:
This is depending on arrow-array 35 despite the crate as a whole still depending on arrow 34. This is likely to cause confusion

Reply:
Fix in https://github.com/apache/arrow-datafusion/pull/5724